CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/algorithms/images/Mat.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
```

Comment:
Does it make sense to rename the Mat class to `Image` or `LoadedImage` or is it as generically usable as the OpenCV Mat where this name probably comes from right?

Reply:
Our idea was to make it an equivalent to the Mat class of OpenCV, but the class is way more complex than we could manage to implement. Right now it basically just wraps the byte array into a more 2d matrix equivalent, which would only really be needed for working on image data, If you prefer that we can rename it. But right now this just serves as a demonstration purpose and has no real use case yet.